drivers/counter/counter_ll_stm32_timer.c

```diff
@@ -655,6 +772,8 @@ void counter_stm32_irq_handler(const struct device *dev)
 	BUILD_ASSERT(NUM_CH(TIM(idx)) <= TIMER_MAX_CH,				  \
 		     "TIMER too many channels");				  \
 										  \
+	IF_ENABLED(CONFIG_COUNTER_CAPTURE,					  \
+		(PINCTRL_DT_INST_DEFINE(idx);))					  \
```

Comment:
```suggestion
	IF_ENABLED(CONFIG_COUNTER_CAPTURE, (PINCTRL_DT_INST_DEFINE(idx);))	  \
										  \
```
(with an escaped empty line below)